templates/CRM/Mailing/Form/Search.tpl

```diff
@@ -26,8 +26,8 @@
 <div class="crm-block crm-form-block crm-search-form-block">
   <table class="form-layout">
     <tr>
-       <td>{$form.mailing_name.label} {help id="id-mailing_name"}<br />
-        {$form.mailing_name.html|crmAddClass:big}
+       <td>{$form.name.label} {help id="id-mailing_name"}<br />
```

Comment:
@seamuslee001 gave this MOP - https://github.com/civicrm/civicrm-core/pull/15652 - won't need this when that is done